plugins/inputs/ipmi_sensor/ipmi.go

```diff
@@ -65,6 +69,12 @@ var sampleConfig = `
 
   ## Schema Version: (Optional, defaults to version 1)
   metric_version = 2
+
+  ## If ipmitool should use a cache
+  use_cache = true
+
+  ## Path to the ipmitools cache file (defaults to OS temp dir)
+  cache_path = "/tmp"
```

Comment:
done with 7dab887
I was under the impression everyone knows what /tmp means while not everyone may know what strange path windows uses. Especially since it was made clear in the comment what to expect.

Reply:
Not these lines look like this:
`cache_path = "/tmp"	  # cache_path = ""`

Reply:
whoopsie, fixed